### DISCUSSION TITLE
Welcome to EH Forwarder Bot Discussions! 欢迎来到 EH Forwarder Bot 社区！

### DISCUSSION THREAD

**Discussion by blueset** [blueset/ehForwarderBot#236] [General 一般讨论]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  As for the current status of the EFB community, we are using both English and Chinese as the common languages of the forum. You are welcomed to use either of the langauges here, but it’ll be great if you can write in both!

  If you have created something for EFB, be it a channel, a middleware, a docker image, a tutorial, you are more than welcomed to post them here. If your work would fit into a GitHub repository, you will also be invited to join this organization to work together!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 欢迎!
  我们启用了 GitHub Discussions 来作为一个社区成员间联系交流的平台。在这里，你可以：
  * 提出你想知道的问题；
  * 分享想法；
  * 与其他社区成员互动；
  * 欢迎他人并保持开放的心态。请记住，这是一个我们共同建立和守护的社区 💪。

  鉴于 EFB 用户群体的现状，我们使用中英双语作为论坛的通用语言。我们欢迎使用其中任何一种语言，但如有余力，希望能够在此同时使用双语。

  如果你有任何 EFB 相关的作品，无论是一个主端、从端、中间件、Docker 容器、镜像、或是教程，都欢迎发到这里。如果你的作品适合放在 GitHub 仓库中，我们也会邀请你加入这个 GitHub Org!

  欢迎你在下面的评论中写上你的自我介绍，并告诉我们你和 EFB 的故事。

**Comment by blueset** [comment]
Hi all! I’m Eana! I’ve been maintaining EFB for quite a few years now. I’m now maintaining the framework, Telegram Master channel (ETM), WeChat Slave channel (EWS), and Facebook Messenger Slave channel (EFMS). It’s really my pleasure to have all of your amazing users and developers in this community. 🎇 Wish you all a happy, healthy and safe new year! 🎆

With love,
Eana

大家好，我是 Eana! 回过头来看我维护 EFB 已经有不少年头了。现在我在维护着 EFB 框架、Telegram 主端（ETM）、微信从端（EWS）和 Facebook Messenger 从端（EFMS）。非常感激在这个社区里遇见了各位真诚友爱的用户和开发者。🎇 希望大家在新的一年里健康快乐，平安幸福！🎆

顺颂时祺
Eana 敬上

**Comment by Kvinam** [comment]
大佬还在更新吗

**Reply by blueset** [reply]
至少人还在（）

**Reply by daybreak-git** [reply]
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/wechatPc/__init__.py", line 79, in event_handler
    await h(event['body'])
  File "/usr/local/lib/python3.8/dist-packages/efb_wechat_pc_slave/__init__.py", line 171, in on_msg_receive
    efb_msg.deliver_to = coordinator.master
AttributeError: module 'ehforwarderbot.coordinator' has no attribute 'master'
```

大佬这个报错是什么问题啊

**Reply by blueset** [reply]
@daybreak-git 
EFB 框架会先初始化从端，后初始化主端。所有初始化完成之后协调器会调用各信道的 `poll()` 方法开始消息处理。
上面的错误是在 `poll()` 方法被调用之前开始处理消息导致的。此时主端仍没有开始初始化。

**Reply by iTeraxo** [reply]
> @daybreak-git EFB 框架会先初始化从端，后初始化主端。所有初始化完成之后协调器会调用各信道的 `poll()` 方法开始消息处理。 上面的错误是在 `poll()` 方法被调用之前开始处理消息导致的。此时主端仍没有开始初始化。
请问这个情况该怎么解决？谢谢
Traceback (most recent call last):
  File "/usr/local/bin/ehforwarderbot", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/ehforwarderbot/__main__.py", line 330, in main
    conf = config.load_config()
  File "/usr/local/lib/python3.9/dist-packages/ehforwarderbot/config.py", line 41, in load_config
    raise ValueError(_("Master Channel is not specified in the profile config."))
ValueError: Master Channel is not specified in the profile config.
